manager2/src/app/plugin/plugin.component.ts

```diff
@@ -70,7 +70,18 @@ export class GalaxyPluginComponent extends BasePluginComponent implements OnInit
 @Component({
     template: `
 <div>
-<div class="alert alert-info">Plugin Genostack is active<br><br> A cloud project will be created on user activation, and deleted on user deletion.</div>
+<div *ngIf="loading">Loading...</div>
+<div *ngIf="data.api_status" class ="alert alert-danger">{{data.api_status}}</div>
```

Comment:
should enclode I think with a *ngif="data" , data must be null/undefined/... and test in data.XXX will raise error